states/huntsman/taking_darks.py

```diff
@@ -35,7 +35,7 @@ def on_enter(event_data):
                 pocs.observatory.take_dark_fields(camera_list=cameras,
                                                   exptimes_list=exptimes)
 
-            pocs.next_state = 'parked'
-
     except Exception as e:
         pocs.logger.warning("Problem with preparing: {}".format(e))
+
+    pocs.next_state = 'parked'
```

Comment:
The mount has to be parked to do these darks, so the _previous_ state will be `parked`, but the `next_state` should probably be `housekeeping` (or `sleeping`).

Reply:
Following up on this, that also means that the logic to get **to** the `taking_darks` state should go in `parked`. 

It should be changed to something like:

```py
# Psuedo-code below
if is_dark and weather_is_bad:
    pocs.next_state = 'taking_darks'
else:
    pocs.next_state = 'housekeeping'
```

It might be more complex than that, depending.

Alternatively, we could consider the act of taking darks to be part of `housekeeping` itself, but we probably best to keep it a little separate.